thumbs/pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[project]
+name = "thumbs"
+version = "0.1.0"
```

Comment:
Are we treating `thumbs` as a "package" here?
If not, lets also set

```
[tool.uv]
package = false
```

here and make the version number `0.0.0`

Reply:
Nice, thanks